src/main/constraints/78_webhooks.sql

```diff
@@ -0,0 +1,17 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Foreign key constraint for the user_id field of the webhooks table.
+--
+ALTER TABLE ONLY webhooks
+      ADD  CONSTRAINT user_id_fkey
+      FOREIGN KEY (user_id)
+      REFERENCES users(id);
+
+--
+-- Foreign key constraint for the type_id field of the webhooks table.
+--
+ALTER TABLE ONLY webhooks
+      ADD  CONSTRAINT type_id_fkey
```

Comment:
I thought `CONSTRAINT` names had to be globally unique in the schema (so not just unique for the `webhooks` table). So maybe these constraints could be less generic? Maybe `webhooks_user_id_fkey` and `webhooks_type_id_fkey`?

Reply:
👍